langchain4j/src/test/java/dev/langchain4j/classification/EmbeddingModelTextClassifierTest.java

```diff
@@ -193,11 +193,11 @@ void should_return_one_category_by_default() {
     void should_return_multiple_categories() {
 
         TextClassifier<CustomerServiceCategory> classifier = new EmbeddingModelTextClassifier<>(
-            new AllMiniLmL6V2QuantizedEmbeddingModel(),
-            examples,
-            2,
-            0,
-            0.5
+                new AllMiniLmL6V2QuantizedEmbeddingModel(),
```

Comment:
please revert reformatting everywhere in this PR

Reply:
@langchain4j , reverted this file. Please reformat all code yourself in a separate PR. Then you will know that there are no other changes, except formatting